docker/jenkins1-centos/Dockerfile

```diff
@@ -1,6 +1,14 @@
 FROM openshift/jenkins-1-centos7
 
+USER root
```

Comment:
[Proposition] If we require root here we should revert to jenkins user `USER 1001` after we execute all required commands. 
Then container user would not be changed.